src/main/scala/mix/kafka/monitor/LibratoOffsetReporter.scala

```diff
@@ -73,3 +65,16 @@ class LibratoOffsetReporter(metrics: MetricRegistry,
     }
   }
 }
+
+object LibratoOffsetReporter {
+  def getMetricName(offsetInfo: OffsetInfo): String = {
+    val nameComponents = Seq (
+      offsetInfo.group,
+      offsetInfo.topic,
+      offsetInfo.partition.toString,
+      "lag"
+    )
+
```

Comment:
Good idea - even if its allowed - i think it is still good to convert everything to lowercase for consistency and readability